code/datums/components/deployable_item.dm

```diff
@@ -82,6 +82,12 @@
 		if(user.do_actions)
 			user.balloon_alert(user, "You are already doing something!")
 			return
+		if(istype(item_to_deploy, /obj/item/weapon/gun/sentry))
+			for(var/turf/turf_to_check AS in RANGE_TURFS(SENTRY_DEPLOY_RANGE_LIMIT, location))
+				for(var/obj/machinery/deployable/mounted/sentry/deployable_sentry in turf_to_check)
+					if(get_dist_euclide(deployable_sentry, item_to_deploy) >= SENTRY_DEPLOY_RANGE_LIMIT)
```

Comment:
doesn't this make you unable to deploy a sentry *unless* another is adjacent?

Reply:
Doesn't seem to be the case!

Reply:
this code feels kinda hacky in general, could this not be part of the sentry code instead of all deployables? also you can use locate(sentry) to get all sentrys instead of checking every turf for a sentry manualy